File: final-evaluation-generated-code/test-2/gemini.js
```javascript
async function invokeSubtract(value) {
  const url = 'http://localhost:3000/http-express-calculator-simple/actions/subtract';
  const options = {
    method: 'POST',
    headers: {
      'Content-Type': 'application/json'
    },
    body: JSON.stringify({ input: value })
  };

  try {
    const response = await fetch(url, options);
    if (!response.ok) {
      throw new Error(`HTTP error! status: ${response.status}`);
    }
    const data = await response.json();
    return data.output;
  } catch (error) {
    console.error('Error invoking subtract action:', error);
    throw error;
  }
}


```

File: final-evaluation-generated-code/test-8/gemini.py
```python
import requests
import json

def subtract(value):
    url = "http://localhost:3000/http-express-calculator-simple/actions/subtract"
    headers = {'Content-Type': 'application/json'}
    payload = {"input": value}
    response = requests.post(url, headers=headers, data=json.dumps(payload))
    if response.status_code == 200:
        return response.json()["output"]
    else:
        return None

#Example
result = subtract(5)
print(result)

```